tests/class_file.rs

```diff
@@ -1,11 +1,17 @@
 use crate::class_file::ClassFileVersion;
+use rlass::class_file::ClassFileVersion;
 
 #[test]
 fn version_supports_test() {
-    let first = ClassFileVersion::latest();
-    let second = ClassFileVersion::new(52, 0);
+    let first = ClassFileVersion::new(53, 0);
+    let second = ClassFileVersion::new(52, 1);
+    let third = ClassFileVersion::new(52, 0);
 
     assert!(first.supports(second));
+    assert!(first.supports(third));
+    assert!(second.supports(third));
     assert_ne!(second.supports(first));
```

Comment:
`assert_ne!` won't work here, because it checks if two objects are **not** the same. You have to replace it with `assert!(!condition)`

Reply:
I was wondering, because i think this code was already there before. Anyway, I'll change all those.